src/module/linux/linux.h

```diff
@@ -1,9 +1,16 @@
 #pragma once
 // Linux specific definitions
 #include "syscalls.h"
+#include "netlink.h"
 
 #define KRF_SAFE_WRITE(x) KRF_CR0_WRITE_UNLOCK(x)
-#define KRF_LOG(...) printk(KERN_INFO __VA_ARGS__)
+#define KRF_LOG(...)                                                                               \
+  ({                                                                                               \
+    char buf[KRF_NETLINK_BUF_SIZE]; /* max message size */                                         \
```

Comment:
This might lead to some confusing errors when expanded in functions with a `buf` already declared, and probably doesn't need to be independently stack-allocated each time (since `snprintf` will correctly NULL-terminate for us). I think we can declare it as `krf_msg_buf` somewhere and provide an `extern` like we do for our config state.